guides/hosting/performance/performance-tweaks.md

```diff
@@ -164,19 +199,21 @@ opcache.preload_user=nginx
 
 ## .env.local.php
 
-[Symfony recommends](https://symfony.com/doc/current/configuration.html#configuring-environment-variables-in-production) that a `.env.local.php` file is used in Production instead of a `.env` file to skip parsing of the  .env file on every request.
+[Symfony recommends](https://symfony.com/doc/current/configuration.html#configuring-environment-variables-in-production) that a `.env.local.php` file is used in Production instead of a `.env` file to skip parsing of the .env file on every request.
```

Comment:
```suggestion
[Symfony recommends](https://symfony.com/doc/current/configuration.html#configuring-environment-variables-in-production) that a `.env.local.php` file is used in production instead of a `.env` file to skip parsing of the `.env` file on every request.
```